cmd/mysterium_server/mysterium_server.go

```diff
@@ -58,22 +57,128 @@ func main() {
 		fmt.Println(versionSummary)
 		fmt.Println()
 
-		log.Infof("User agreed with terms & conditions: %v", options.AgreedTermsConditions)
-		runCMD(options)
+		fmt.Printf("User agreed with terms & conditions: %v\n", options.AgreedTermsConditions)
+		runCMD(options.NodeOptions, options.ServiceOptions)
 	}
 }
 
-func runCMD(options server.CommandOptions) {
-	serverCommand := server.NewCommand(options)
+type commandOptions struct {
+	Version               bool
+	LicenseWarranty       bool
+	LicenseConditions     bool
+	AgreedTermsConditions bool
 
-	if err := serverCommand.Start(); err != nil {
+	NodeOptions    node.Options
+	ServiceOptions service.Options
+}
+
+func parseArguments(args []string) (options commandOptions, err error) {
+	flags := flag.NewFlagSet(args[0], flag.ContinueOnError)
+
+	err = cmd.ParseDirectoryArguments(flags, &options.NodeOptions.Directories)
+	if err != nil {
+		return
+	}
+
+	flags.StringVar(
+		&options.NodeOptions.OpenvpnBinary,
+		"openvpn.binary",
```

Comment:
Do `mysterium_node service` accepts` openvpn.binary` parameter?

Reply:
Yes, it's passed in global variables:
```
mysterium_node --openvpn.binary=openvpn service --openvpn.protocol=tcp --openvpn.port=1111
```